### ISSUE TITLE
FreeBSD checks fail consistently

### ISSUE THREAD

**Issue by szubersk** [openzfs/zfs#13036]
### System information
GitHub workflows and builbot

### Describe the problem you're observing
Two issues really:
1. the tag `Required` was removed from all checks that previously had it
2. FreeBSD checks now fail consistently
```
Tests with results other than PASS that are unexpected:
    FAIL alloc_class/alloc_class_012_pos (expected PASS)
    FAIL cli_root/zpool_events/zpool_events_clear (expected PASS)
```

Most likely (1) lead to sneaking in (2).

Tagging @behlendorf, the CI lead.

**Comment by freqlabs** [comment]
Fixed by https://github.com/openzfs/zfs/pull/13032

**Comment by szubersk** [comment]
> Fixed by #13032
While (2) is fixed, I'm afraid issue (1) is still there. Something happened to checks configuration in the project.
